interface/wx/httprequest.h

```diff
@@ -0,0 +1,232 @@
+/////////////////////////////////////////////////////////////////////////////
+// Name:        httprequest.h
+// Created:     2018-10-14
+// Copyright:   (c) 2018 wxWidgets development team
+// Licence:     wxWindows licence
+/////////////////////////////////////////////////////////////////////////////
+
+/**
+    @class wxHTTPRequest
+
+    wxHTTPRequest allows for simple HTTP requests using the operating systems
+    components as implementation. The latest features of their operating
+    system will be used if available.
+
+    This class is well suited for simple HTTP requests.
+
+    For example:
+
+    @code
+        wxHTTPRequest request("http://www.wxwidgets.org/downloads/logos/blocks.png");
+        if (request.Send())
+        {
+            // Write the response data to file
+            wxInputStream* inputStream = request->GetResponseStream();
+            wxFileOutputStream outputStream("blocks.png");
```

Comment:
Could be even better/simpler/more illustrative to just pass the stream to `wxImage` ctor.